SaveAllTheTime/source.extension.vsixmanifest

```diff
@@ -1,22 +1,22 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
-  <Metadata>
-    <Identity Id="SaveAllTheTime..e5079769-4785-4c30-88d0-6d570504b964" Version="1.3.2" Language="en-US" Publisher="Paul Betts" />
-    <DisplayName>SaveAllTheTime</DisplayName>
-    <Description xml:space="preserve">SaveAllTheTime makes it so you never commit files to Git without saving ever again. It also reminds you to commit often by showing you a widget at the bottom right corner of your screen</Description>
-    <MoreInfo>http://paulcbetts.github.io/SaveAllTheTime</MoreInfo>
-    <License>COPYING.txt</License>
-    <ReleaseNotes>CHANGELOG.txt</ReleaseNotes>
-    <Icon>media-floppy.png</Icon>
-    <Tags>git</Tags>
-  </Metadata>
-  <Installation>
-    <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[11.0)" />
-  </Installation>
-  <Dependencies>
-    <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="4.5" />
-  </Dependencies>
-  <Assets>
-    <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="%CurrentProject%" Path="|%CurrentProject%|" />
-  </Assets>
+    <Metadata>
+        <Identity Id="SaveAllTheTime..e5079769-4785-4c30-88d0-6d570504b964" Version="1.3.2.1" Language="en-US" Publisher="Paul Betts" />
+        <DisplayName>SaveAllTheTime</DisplayName>
+        <Description xml:space="preserve">SaveAllTheTime makes it so you never commit files to Git without saving ever again. It also reminds you to commit often by showing you a widget at the bottom right corner of your screen</Description>
+        <MoreInfo>http://paulcbetts.github.io/SaveAllTheTime</MoreInfo>
+        <License>COPYING.txt</License>
+        <ReleaseNotes>CHANGELOG.txt</ReleaseNotes>
+        <Icon>media-floppy.png</Icon>
+        <Tags>git</Tags>
+    </Metadata>
+    <Installation>
+        <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[11.0,14.0]" />
```

Comment:
It's enough to put `[11.0,]` here, without specifying `14.0` explicitly. This way it will be installed into any version *starting with* 11.0 (VS2012)